archive/common/src/main/scala/uk/ac/wellcome/platform/archive/common/progress/models/Progress.scala

```diff
@@ -19,25 +19,25 @@ case class Progress(id: UUID,
 case object Progress extends URIConverters {
   sealed trait Status
 
-  private val initialisedString = "initialised"
+  private val acceptedString = "accepted"
   private val processingString = "processing"
-  private val successString = "success"
-  private val failureString = "failure"
+  private val succeededString = "succeeded"
+  private val failedString = "failed"
 
-  case object Initialised extends Status {
-    override def toString: String = initialisedString
+  case object Accepted extends Status {
+    override def toString: String = acceptedString
```

Comment:
Will this break when deserialising existing stored progress?  As it is progress maybe that's fine...

Reply:
Yes.

I thought the database was only test data, so that was safe, but perhaps I've missed something?

Reply:
I think then something to be aware of.